pallets/registrar/src/lib.rs

```diff
@@ -184,13 +182,8 @@ pub mod pallet {
     >;
 
     #[pallet::storage]
-    pub type ParaGenesisData<T: Config> = StorageMap<
-        _,
-        Blake2_128Concat,
-        ParaId,
-        ContainerChainGenesisData<T::MaxLengthTokenSymbol>,
-        OptionQuery,
-    >;
+    pub type ParaGenesisData<T: Config> =
+        StorageMap<_, Blake2_128Concat, ParaId, ContainerChainGenesisData, OptionQuery>;
```

Comment:
there is no migration here right? the values are the same

Reply:
I used the value in the runtime as the new constant limit.
Also IIRC the encoded data doesn't contain the limit.

Reply:
> the encoded data doesn't contain the limit

Correct, and it would only need a migration if you decrease the limit